playground/frontend/web/index.html

```diff
@@ -37,7 +37,9 @@
   <base href="/">
 
   <meta content="IE=Edge" http-equiv="X-UA-Compatible">
-  <meta name="description" content="Apache Beam Playground">
+  <meta name="description"
+        content="Apache Beam is an open source, unified model for defining both batch and streaming data-parallel processing pipelines. Apache Beam Playground is an interactive environment to try out Beam transforms and code examples without having to install Apache Beam in your environment. Beam Playground support Java, Python, Go SDKs, and SCIO and had a catalog of hundreds of code examples to try."/>
```

Comment:
May we remove: `Beam Playground support Java, Python, Go SDKs, and SCIO and had a catalog of hundreds of code examples to try.`.  As new languages are added, one would need to remember to update this list.